Code/GraphMol/MolPickler.h

```diff
@@ -45,12 +45,12 @@ class MolPicklerException : public std::exception {
 namespace PicklerOps {
 typedef enum {
   NoProps = 0,     // no data pickled
-  MolProps = 0x1,  // only public non computed properties
-  AtomProps = 0x10,
-  BondProps = 0x100,
-  QueryAtomData = 0x100,
-  PrivateProps = 0x10000,
-  ComputedProps = 0x100000,
+  MolProps      = BOOST_BINARY(1),  // only public non computed properties
+  AtomProps     = BOOST_BINARY(10),
```

Comment:
Doesn't this block of changes end up breaking backwards compatibility (i.e. old pickles will no longer be readable)?

Reply:
I don't think so, these values are not written to the stream, the Tags::enum are.

Never the less, I'll make a regression test!

Reply:
Ah, ok, now I see how this is done.
Having a regression test that loads a molecule serialized with the current version is probably still a good idea though (I guess that's what you're thinking of).